code/data-pipeline/tests/test_load_data.py

```diff
@@ -0,0 +1,17 @@
+from data_pipeline.loader import FluDataLoader
+
+def test_load_data_sources():
+    fdl = FluDataLoader('../../data-raw')
```

Comment:
This is probably bad form as it's referring to a `data-raw` folder that sits outside of the folder structure for the `data-pipeline` module and the tests folder in particular...  However, doing anything else would take a lot of effort and I kind of believe that maybe `data-raw` and `data-pipeline` should be more tightly coupled (and separate from the flusion repo) anyways.

Reply:
yeah, it's definitely a footgun, but your instructions for running the test suite already state that you need to be in a specific working directory

if it really bothers you, the other thing you could do is copy a subset of the `data-raw` stuff into a folder under the `test` directory and point the `FluDataLoader` there.